src/Raven.Studio/typescript/viewmodels/database/patch/patchTester.ts

```diff
@@ -233,11 +254,47 @@ class patchTester extends viewModelBase {
     previewDocument() {
         const spinner = this.spinners.preview;
         const documentId: KnockoutObservable<string> = this.documentId;
-        const documentIdValidationGroup = this.validationGroup;
+        const documentIdValidationGroup = this.testDocumentValidationGroup;
         const db = this.activeDatabase;
 
         documentPreviewer.preview(documentId, db, documentIdValidationGroup, spinner);
     }
+
+    getLoadedCount(): KnockoutComputed<string> {
```

Comment:
those 3 functions are bad. Each invocation produces own observable and we calculate this few times. Move this to class field.

Reply:
done